ledger/ledger-api-test-tool-on-canton/bootstrap.canton

```diff
@@ -1,5 +1,7 @@
 nodes.local.start
 
+test_domain.service.set_mediator_deduplication_timeout(6.minutes)
```

Comment:
I just duplicate what was in bootstrap-dev.canton

Reply:
👍